pkg/backporter/backporter_test.go

```diff
@@ -219,17 +219,19 @@ func TestCreateCloneHandler(t *testing.T) {
 	if err != nil {
 		t.Fatalf("error getting bug details from Fake.")
 	}
+	toBeCloned.TargetRelease = []string{"4.5.0"}
 	expectedCloneID := toBeClonedID + 1
 	expectedClone := *toBeCloned
 	expectedClone.ID = expectedCloneID
 	expectedClone.TargetRelease = []string{clonedRelease}
 	testcases := []struct {
```

Comment:
We should somehow verify the new functionality?

Reply:
This test should verify the new functionality, since the child relation would be updated in the Blocks and DependsOn fields. But I guess I could add a testcase to verify that multiple clones are created as well when necessary